cmd/lifecycle/restorer.go

```diff
@@ -42,6 +42,7 @@ func (r *restoreCmd) DefineFlags() {
 		cli.FlagGeneratedDir(&r.GeneratedDir)
 		cli.FlagUseLayout(&r.UseLayout)
 		cli.FlagLayoutDir(&r.LayoutDir)
+		cli.FlagInsecureRegistries(&r.InsecureRegistries)
```

Comment:
Is it possible to move this into a `if r.PlatformAPI.AtLeast("0.13")` block?

Reply:
I moved all the `cli.FlagInsecureRegistries(&r.InsecureRegistries)` from the 0.12 guard to the 0.13 ones and added this too, thanks 😄